### ISSUE TITLE
java.lang.ClassNotFoundException: org.springframework.boot.context.embedded.FilterRegistrationBean

### ISSUE THREAD

**Issue by ivorzhou** [Unicon/cas-client-autoconfig-support#8]
When I use the spring boot version 1.5.2.RELEASE
the class FilterRegistrationBean has move to 
org.springframework.boot.web.servlet.FilterRegistrationBean

**Comment by junxy** [comment]
This problem can be solved. 

1. Download the code and then compile the version. (current 1.4.0-GA-SNAPSHOT)
2. Use compiled local version
```
<dependency>
    <groupId>net.unicon.cas</groupId>
    <artifactId>cas-client-autoconfig-support</artifactId>
    <version>1.4.0-GA-SNAPSHOT</version>
</dependency>
```

**Comment by dima767** [comment]
Addressed in `1.4.0-GA` release
